packages/rocketchat-slashcommands-unflip/unflip.js

```diff
@@ -0,0 +1,20 @@
+/*
+* Unflip is a named function that will replace /unflip commands
+* @param {Object} message - The message object
+*/
+
+
+function Unflip(command, params, item) {
```

Comment:
![Codacy](https://www.codacy.com/assets/images/favicon.png) Issue found: ["Unflip" is defined but never used](https://www.codacy.com/app/RocketChat/Rocket-Chat/file/2657490839/issues/source?bid=3199634&fileBranchId=3328556#l7)